cli/pcluster/utils.py

```diff
@@ -754,3 +754,18 @@ def get_cluster_capacity(stack_name):
         if scheduler == "awsbatch"
         else get_asg_settings(stack_name).get("DesiredCapacity")
     )
+
+
+def get_efa_enabled_instance_types():
+    client = boto3.client("ec2")
+    instance_types = []
+    next_token = ""
+
+    while next_token is not None:
```

Comment:
Shall we use the paginator: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Paginator.DescribeInstanceTypes

Reply:
Thanks, switched to paginate_boto3.